substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/JavaNetSubstitutions.java

```diff
@@ -160,6 +171,34 @@ static URLStreamHandler getURLStreamHandler(String protocol) {
         return result;
     }
 
+    static URLStreamHandler createResourcesURLStreamHandler() {
+        URLStreamHandler answer = new URLStreamHandler() {
+            @Override
+            protected URLConnection openConnection(URL url) throws IOException {
+                return new URLConnection(url) {
+                    @Override
+                    public void connect() throws IOException {
+                    }
+
+                    @Override
+                    public InputStream getInputStream() throws IOException {
+                        Resources.ResourcesSupport support = ImageSingletons.lookup(Resources.ResourcesSupport.class);
+                        // remove "protcol:" from url to get the resource name
+                        String resName = url.toString().substring(1+JavaNetSubstitutions.RESOURCE_PROTOCOL.length());
+                        final List<byte[]> bytes = support.resources.get(resName);
+                        if (bytes == null || bytes.size() < 1) {
+                            System.err.println("Couldn't find resource "+resName);
```

Comment:
Is this line required by the spec or the original implementation? I would like to get rid of it as it fails our checks for not using `System.err` here.